sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BaseBlobClientBuilder.java

```diff
@@ -46,7 +47,7 @@ public T customerProvidedKey(CustomerProvidedKey key) {
      * @return the {@code UserAgentPolicy} that will be used in the {@link HttpPipeline}.
      */
     @Override
-    protected final UserAgentPolicy getUserAgentPolicy() {
+    protected UserAgentPolicy getUserAgentPolicy() {
```

Comment:
This cant be final since we want blob-cryptography's user agent policy to say azure-storage-blob-cryptography instead of azure-storage-blob